client/disputes/evidence/index.js

```diff
@@ -109,7 +110,7 @@ export const DisputeEvidenceForm = ( props ) => {
 			onChange( field.key, value );
 		},
 		disabled: readOnly,
-		help: expandHelp( field.description ),
+		help: readOnly ? '' : expandHelp( field.description ),
```

Comment:
IMO, help tooltips should remain even if we are in read-only mode.

Reply:
My intention was to keep the submitted dispute details page cleaner, by removing all helper text. By helper text, I mean this :

![image](https://user-images.githubusercontent.com/4162931/194838473-13ce62a4-8129-4d24-a983-796f5769a4bf.png)

Since the merchant would've already read the instruction in that and submitted details, I thought there is not much utility for that text after submission.  Do you think it will be still helpful to have it, @kalessil ?

Reply:
This might be helpful, thanks for sharing your reasoning. Let's check with @rogermattic (who is our UI/UX guru) on keeping/hiding help text for disabled evidence upload.

Reply:
I agree with @kalessil's point here about keeping the hint texts visible even on submitted forms for (a) consistency, and (b) later reference. But we'll wait for @rogermattic's final call.

Reply:
Maybe removing the verbs like `Provide` will help in the read only version. Although users would obviously be seeing the prominent message on top mentioning that the form is read-only. Will await the call from UI/ UX colleague(s) :) .

Reply:
Reply from [UI / UX team](https://github.com/Automattic/woocommerce-payments/issues/495#issuecomment-1284366514) . Linking here for posterity.

Reply:
@nagpai @kalessil  sorry for the late reply! I'm just looking at it now. I agree with what  @bradyv mentioned in [this other thread](https://github.com/Automattic/woocommerce-payments/issues/495#issuecomment-1284366514) 
and wondering if we could have the helper text visible in case the form is in the edit mode and then hidden, for simplicity when it's all submitted? Or would that be a too complex solution to implement?

Reply:
@rogermattic - No worries :). The PR in its current form, does hide the helper text once submitted. The helper text remains visible before submission, when editing is possible. Thanks to Brad and you for confirming this 👍

Reply:
Awesome:) thanks!

Reply:
I am marking this conversation as resolved, since we have received recommendations from our colleagues in the UI / UX team ( Magda and Brad ) . Thanks for the guidance, @kalessil  and @anu-rock 🙌